File: FizzBuzz.js
function fizzBuzz(num){
	/**var num = prompt('Please enter a number: ');*/
	/** checks number is divisible by 3 and 5*/
	if (num%15 == false){

		msg = 'FizzBuzz';
	/** checks number is divisible by 5*/
	}else if (num%5 == false){

		msg = 'Buzz';
	/** checks number is divisible by 3*/
	}else if (num%3 == false){

		msg = 'Fizz';
	/**number not divisible by either 3 or 5*/
	}else{

		msg = num;
	}

	return msg;
}

File: fizz_buzz.py
def fizz_buzz(num):
    #checks number is divisible by 3 and 5*
    if (num%15 == False):
        msg = 'FizzBuzz'
    #checks number is divisible by 5*   
    elif (num%5 == False):
        msg = 'Buzz'
    #checks number is divisible by 3 
    elif (num%3 == False):
        msg = 'Fizz'
    #number not divisible by either 3 or 5*
    else:
        msg = num
    return msg     
 